doc/tutorials/run_all_tests_with_docker.md

```diff
@@ -66,16 +81,7 @@ So from your root project folder run the following:
 docker run -it --rm \
 -v "$PWD:/home/jenkins/workspace" \
 -w /home/jenkins/workspace \
-hub-public.libelektra.org/<image_name>:<tag_name>
-```
-
-Example:
-
-```sh
```

Comment:
Why do you remove the example?

Reply:
It showed how to run a container created from a public image. Since we do not recommend it, I thought it is not needed anymore.

Reply:
I thought you will simply leave this part and say it is not recommended.